website/templates/contact/contact.html

```diff
@@ -0,0 +1,96 @@
+{% extends "base.html" %}
+{% load staticfiles %}
+{% load wagtailcore_tags %}
+{% load wagtailcore_tags snippets %}
+
+{% block title %}Contact Us {% endblock %}
+{% block navlinks %}
+   {% with 'communities' as nav %}
+      {{ block.super }}
+   {% endwith %}
+{% endblock %}
+
+{% block content %}
+<div class="banner-container">
+    <div class="banner-element">
+        Help support open science today.
+        <a href="/donate/" class="btn banner-button" onClick="ga('send', 'event', 'marketing', 'click', 'COS Donate Banner')">Donate Now</a>
+    </div>
+</div>
+
+<div class="page-container">
+    <!-- BEGIN CONTAINER -->
+    <div class="container min-height margin-top-50">
+        <!-- BEGIN CONTENT -->
+        <div class="community-content">
+            <div class="margin-top-20 page-header">
+                <h1>
+                    <center>
+                        <b>Contact Us</b><br>
+                        <small style="font-size:20px;">Want to find out more about getting involved?  We'd love to hear from you and discuss more about your interests!</small>
+            	    </center>
+            	</h1>
+            </div>
+            <div>
+                <div class="form-horizontal">
+                    <form method="post" class="contact-form">
+                        {% csrf_token %}
+                        <input type=hidden name="oid" value="00Dj0000001p7Or">
```

Comment:
This might be the ID of the backend form we are submitting to, would it be worth making this as setting as well?

Reply:
done @icereval